data/test_validate/domain.yml

```diff
@@ -0,0 +1,21 @@
+intents:
+  - greet
+  - goodbye
```

Comment:
You can use the `default_domain` from conftest by passing it as an argument to one of your validate test functions, please remove this file